src/http_req.c

```diff
@@ -172,9 +176,18 @@ const char* um_http_resp_header(um_http_resp_t *resp, const char *name) {
 static int http_headers_complete_cb(http_parser *p) {
     um_http_req_t *req = p->data;
     req->state = headers_received;
+
+    const char *compression = um_http_resp_header(&req->resp, "content-encoding");
+    if (compression) {
+        set_http_header(&req->resp.headers, "Content-Length", NULL);
+        set_http_header(&req->resp.headers, "transfer-encoding", "chunked");
```

Comment:
it's noticeable to me since they are so close that this is "Content-Length" vs "transfer-encoding". Why not "Transfer-Encoding" and keep it consistent please? :)